server/src/application/application.service.ts

```diff
@@ -22,6 +22,18 @@ export class ApplicationService {
       });
     }
 
+    if (query.confirmationId) {
+      queryBuilder.andWhere('app.confirmationId ILIKE :confirmationId', {
```

Comment:
They search for exact matches in terms of confirmation number. So we can use =

Reply:
Thank you! Was not sure if we wanted to allow them to find some partially matching confirmation ID's, in case they forget the full ones, but makes sense. Changed to =